pages/linux/dd.md

```diff
@@ -17,7 +17,7 @@
 
 - Benchmark the write performance of a disk:
 
-`dd bs={{1M}} count={{1000000}} if=/dev/zero of={{path/to/file_1GB}}`
+`dd bs={{1024}} count={{1000000}} if=/dev/zero of={{path/to/file_1GB}}`
```

Comment:
```suggestion
`dd bs={{1M}} count={{1024}} if=/dev/zero of={{path/to/file_1GB}}`
```

I think we keep the `M` and change the count. Less zeroes makes it easy to work with. And the other examples use `M` as well.